### ISSUE TITLE
Trailing comma detection broken for ternary operator in parens

### ISSUE THREAD

**Issue by ptarjan** [eslint/eslint#4200]
When configured with:

```
    "comma-dangle": [2, "always-multiline"],
```

This code:

```
    var foo = [
      (bar ? baz : qux),
    ];
```

complains with

```
79 col 18 error| Unexpected trailing comma. (comma-dangle) [javascript/eslint]
```

but if I change the code to

```
    var foo = [
      bar ? baz : qux,
    ];
```

It doesn't complain (correctly).

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
What version are you using?

**Comment by ptarjan** [comment]
@nzakas 1.7.2

**Comment by hzoo** [comment]
Seemed like @nzakas already did a fix for at a similar issue https://github.com/eslint/eslint/pull/4198/files

**Comment by nzakas** [comment]
Yeah, I'm a bit surprised that fix didn't catch this issue.

**Comment by nzakas** [comment]
Working on this.
